CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+New features:
+
+- Add `typing_extensions.Reader` and `typing_extensions.Writer`. Patch by
+  Sebastian Rittau.
+
+Bugfixes:
+
 - Drop support for Python 3.8 (including PyPy-3.8). Patch by [Victorien Plot](https://github.com/Viicos).
 - Add support for inline typed dictionaries ([PEP 764](https://peps.python.org/pep-0764/)).
   Patch by [Victorien Plot](https://github.com/Viicos).
```

Comment:
Neither of these are really bugfixes -- inline TypedDicts is a feature; dropping support for Python 3.8 is an "other changes" kind of thing IMO

Reply:
Ah yes, this is a merge artifact. Will fix.